packages/playwright/src/executors/playwright/schema.json

```diff
@@ -0,0 +1,156 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "version": 2,
+  "title": "Playwright executor",
+  "description": "",
+  "type": "object",
+  "properties": {
```

Comment:
unsure if we should add properties for different sub commands like 
show-report, codegen or make them separate executors?

I was thinking probably both

```shell
nx e2e demo-app --show-report # makes sense as an option here
```
```shell
nx codegen demo-app # makes sense as separate executor.  
```

Could be a follow up to this executor before the release.

Reply:
I think a separate target makes sense, so it's not confusing when users are used to running both `show-report` and `codegen` as separate commands. It removes any potential confusion from actual `playwright test` options.

I'm not sure if a new executor is needed, or can we just use `command` in the generator? like

```jsonc
{
  "codegen": {
    "command": "playwright codegen"
    "options": { "cwd": "<project-root>" }
  }
}
```

There are also commands that seem like they are unrelated to tests (e.g. `open, `cr`, `ff`, etc.). I thnk we can skip those.